spec/02-integration/04-admin_api/02-kong_routes_spec.lua

```diff
@@ -206,6 +206,28 @@ describe("Admin API - Kong routes with strategy #" .. strategy, function()
       assert.is_number(json.server.connections_writing)
       assert.is_number(json.server.connections_waiting)
       assert.is_number(json.server.total_requests)
+      assert.is_nil(json.configuration_hash) -- no config hash in DB mode
+    end)
+
+    it("returns a configuration_hash in dbless mode #off", function()
+      local res = assert(client:send {
+        method = "GET",
+        path = "/status"
+      })
+      local body = assert.res_status(200, res)
+      local json = cjson.decode(body)
+      assert.is_table(json.database)
+      assert.is_table(json.server)
+      assert.is_boolean(json.database.reachable)
+      assert.is_number(json.server.connections_accepted)
+      assert.is_number(json.server.connections_active)
+      assert.is_number(json.server.connections_handled)
+      assert.is_number(json.server.connections_reading)
+      assert.is_number(json.server.connections_writing)
+      assert.is_number(json.server.connections_waiting)
+      assert.is_number(json.server.total_requests)
+      assert.is_string(json.configuration_hash)
+      assert.equal("no_configuration_yet", json.configuration_hash)
```

Comment:
We should make sure that this endpoint is available and works as expected on status_listen listeners as well.

Reply:
Added (and succeeding) with ed05a1de3eee52c091a492bef704e331a23120e5 :+1: